src/main/kotlin/com/cognifide/gradle/aem/tooling/tail/LogAnalyzer.kt

```diff
@@ -0,0 +1,81 @@
+package com.cognifide.gradle.aem.tooling.tail
+
+import com.cognifide.gradle.aem.tooling.tail.io.LogFiles
+import com.cognifide.gradle.aem.tooling.tasks.Tail
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.ObsoleteCoroutinesApi
+import kotlinx.coroutines.channels.Channel
+import kotlinx.coroutines.channels.ReceiveChannel
+import kotlinx.coroutines.channels.SendChannel
+import kotlinx.coroutines.channels.consumeEach
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+
+@UseExperimental(ObsoleteCoroutinesApi::class)
```

Comment:
please add doc with link https://github.com/Kotlin/kotlinx.coroutines/issues/632#issuecomment-425408865
to not forget why this annotation is here / to know that channels API will change ; introduce `TODO` for better visibility

Reply:
OK